testSetupFile.js

```diff
@@ -5,13 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-const jasmineReporters = require('jasmine-reporters');
-
 // Some of the `jest-runtime` tests are very slow and cause
 // timeouts on travis
 jest.setTimeout(70000);
-
-if (global.jasmine && process.env.APPVEYOR_API_URL) {
-  // Running on AppVeyor, add the custom reporter.
-  jasmine.getEnv().addReporter(new jasmineReporters.AppVeyorReporter());
```

Comment:
Too bad Travis doesn't have any concept of test result reporters. This is actually very useful as it pulls out the test results into a separate table view rather than having to look through pages of console output.

![image](https://user-images.githubusercontent.com/91933/47175157-4ef7fd00-d2c7-11e8-8da9-72bd804ceeba.png)